bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerStorage.java

```diff
@@ -181,4 +181,20 @@ default LedgerStorage getUnderlyingLedgerStorage() {
     default void forceGC() {
         return;
     }
+
+    /**
+     * Whether force triggered Garbage Collection is running or not.
+     *
+     * @return
+     *      true  -- force triggered Garbage Collection is running,
+     *      false -- force triggered Garbage Collection is not running
+     */
+    default boolean isInForceGC() {
+        return false;
+    }
+
+
+    default GarbageCollectionStatus getGarbageCollectionStatus() {
+        return null;
```

Comment:
If you change to return `List<GarbageCollectionStatus>`, you can return `Collections.emptyList()`.